src/API/FileController.ts

```diff
@@ -0,0 +1,94 @@
+import * as FormData from 'form-data'
+import * as fs from 'fs'
+import { inject, injectable } from 'inversify'
+import { Collection, Db } from 'mongodb'
+import fetch, {Response} from 'node-fetch'
+import * as Pino from 'pino'
+import { map } from 'ramda'
+
+import { childWithFileName } from 'Helpers/Logging'
+import { minutesToMiliseconds  } from 'Helpers/Time'
+import { asyncPipe } from 'Helpers/asyncPipe'
+
+interface IPFSFileResponseJson {
+  readonly Hash: string
+}
+
+interface FileResponseJson {
+  readonly hash: string
+}
+
+interface File {
+  readonly path: string
+}
+
+enum LogTypes {
```

Comment:
This feels out of place in the FileController. Can you put it in Interfaces?

Reply:
Yeah, that might be another good spot. I plan on refactoring this at some point, I am still experimenting a bit with this and the log method. I might be creating a method creator helper, so I was thinking about sticking the types with the method helper